strr-examiner-web/app/stores/examiner.ts

```diff
@@ -89,6 +110,21 @@ export const useExaminerStore = defineStore('strr/examiner-store', () => {
     return resp
   }
 
+  /**
+   * Send a Notice of Consideration for the specified application.
+   *
+   * @param {string} applicationNumber - The application number.
+   * @param {string} content - The content of the Notice of Consideration.
+   * @returns {Promise<void>}
+   */
+  const sendNoticeOfConsideration = async (applicationNumber: string, content: string): Promise<void> => {
+    await $strrApi(`/applications/${applicationNumber}/notice-of-consideration`, {
+      method: 'POST',
+      body: { content }
+    })
+    nocContent.content = ''
```

Comment:
I would let the called (of this endpoint) to take care of the content clearing. Maybe if it fails, the content should be kept intact to try again.

Reply:
I hooked the content reset within refreshFn being provided to `useExaminerActions` composable.